src/docs/product/issues/issue-details/performance-issues/slow-db-queries/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Slow DB Queries'
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/slow-db-queries/
+description: 'Learn more about Slow DB Query issues and how to diagnose and fix them.'
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Slow DB Query issues are created when a particular `SELECT` SQL query in your application consistently takes longer than `1000ms` to resolve.
+
+## Detection Criteria
+
+The detector for this performance issue looks at singular database spans with the following criteria:
+
+- The span operation starts with `db` and the query is stored within the span description
+- The query is specifically a `SELECT` query
+- The span duration is at least `1000ms`
+
+Keep in mind that an issue will not be created unless the query in question is _consistently_ taking longer than `1000ms`. More specifically, the issue is created only if the same query is detected as a slow query
+in 100 events within a 24 hour period.
+
+## Span Evidence
+
+The "Span Evidence" section of a Slow DB Query issue contains the following fields:
+
+- Transaction name
+- Slow DB Query
+- Duration impact - The duration of the query with respect to the entire transaction, to put into perspective how much of the transaction is being taken up by the query
+
+![Slow DB Query span evidence](span-evidence.png)
+
+View it by going to the **Issues** page in Sentry, selecting your project, clicking on the Slow DB Query issue you want to examine, then scrolling down to the "Span Evidence" section in the "Details" tab.
+
+## Fixing Slow DB Queries
+
+One common cause of slow queries is that the table that is being queried from is un-indexed, so it is recommended to confirm that your table is properly indexed first and foremost. Aside from that, there's no immediately obvious way to fix slow queries, but there are some steps you can take to narrow down what specifically is causing your query to be slow.
```

Comment:
```suggestion

Slow queries often happen when un-indexed tables are queried. To avoid this, make sure that your table is properly indexed. While there aren’t any other obvious fixes, there are steps you can take to narrow down the causes of your query being slow.
```